### DISCUSSION TITLE
requestVerification is not working due to the JWT Token

### DISCUSSION THREAD

**Discussion by teddysbeach** [pocketbase/pocketbase#673] [Q&A] [answered]
Hello, I am trying to use Pocketbase on my personal project,
and I set the SMTP values on the Admin page of the Pocketbase Admin, It works properly as I expected.
but when I receive the email and click on the button [Verify Email], the error comes out

![image](https://user-images.githubusercontent.com/64014774/193245027-bb6fa2a5-4e74-423a-890c-43bd10bb4972.png)

This error happens but I don't know what to do about it!
I set the user.profile API permissions setting but still not working.

I am wondering if this is just an error or my mistake.

* The pocketbase version I am using is pretty low (0.5.1v)!

**Comment by ganigeorgiev** [comment]
Hm, that's strange.

Are you sure that you are sending the request with `client.users.requestVerification()` and not `client.users.requestEmailChange()`?

Could you post here the network response info from your browser's "Inspect Tool > Network" tab. Maybe the above error is not from the requestVerification confirmation call but from something else.

The `requestVerification` api handler doesn't require authentication (see https://github.com/pocketbase/pocketbase/blob/master/apis/user.go#L30).

The user.profile API rules doesn't affect the users management. They apply only to the profiles collection crud actions (list, view, create, update, delete).

**Reply by ganigeorgiev** [reply]
@teddysbeach I cannot reproduce the error. I've just tested it locally and it works as expected. There are no changes to the verification api in the latest release, but you could try upgrading your PocketBase executable and SDK versions and test again (**always make sure to have a backup of your `pb_data` before upgrading!**).

**Comment by teddysbeach** [comment, marked as answer]
My bad, @ganigeorgiev, I found out the reason why it was not working
I am using Nginx to handle the incoming requests for pocketbase, during that, I changed my admin page URL which was (/_/) into (/_admin_/). and the pocketbase's mail action URL was not changed into a new URL.

I've solved the problem by only changing the action URL like below.

<img width="456" alt="image" src="https://user-images.githubusercontent.com/64014774/193256657-f85b7270-ee91-4450-bd21-2cb5aa54d34f.png">


Sorry for wasting your precious time.
I am really happy to use your great output Pocketbase :)

Have a good day!
